views/ctltools/forms/delete-form.hbs

```diff
@@ -1,7 +1,34 @@
 <div class="form-group">
-    <label class="control-label" for="name">
-        Sind Sie sich sicher, dass Sie das folgende Tool löschen wollen?<br>
-        <i>Achtung: Beim Löschen des Tools wird das Tool auch in allen Kursen und Teams dauerhaft gelöscht. Bitte bestätigen Sie, dass das folgende Tool gelöscht werden soll.</i>
+    <fieldset disabled>
+        <label class="control-label" for="name">
+            Sind Sie sich sicher, dass Sie das folgende Tool löschen wollen?
+            <input type="text" style="width: 100%;" name="name" id="name">
```

Comment:
Looks creepy if you dont change the width here but on the other inputs.

Reply:
I discussed this with BA, he wanted it that way